examples/24_solar_battery_grid/README.md

```diff
@@ -0,0 +1,87 @@
+# Example 23: Solar Battery Grid System
+
+## Overview
+
+This example demonstrates a solar + battery + grid system that showcases the new unified grid component. The system can both buy electricity from the grid and sell excess electricity back to the grid using separate grid connection instances.
```

Comment:
"new" will quickly become obsolete, and "unified" won't mean much in this context outside of our dev discussions. I think we should just say "the grid component"